xbmc/video/VideoDatabase.cpp

```diff
@@ -3515,7 +3517,15 @@ void CVideoDatabase::DeleteTvShow(int idTvShow, bool bKeepId /* = false */)
     std::set<int> paths;
     GetPathsForTvShow(idTvShow, paths);
 
-    std::string strSQL=PrepareSQL("SELECT episode.idEpisode FROM episode WHERE episode.idShow=%i",idTvShow);
+    std::string strSQL=PrepareSQL("SELECT episode.idFile FROM episode WHERE episode.idShow=%i",idTvShow);
+    m_pDS2->query(strSQL);
+    while (!m_pDS2->eof())
+    {
+      DeleteStreamDetails(m_pDS2->fv(0).get_asInt());
+      m_pDS2->next();
+    }
```

Comment:
Sorry for being late to the discussion. Can you elaborate why this is necessary here? A few lines below we already call `DeleteEpisode()` which (with your other changes) in turn calls `DeleteStreamDetails()` based on the linked file. So why do we have to do this manually here as well?

Reply:
Nice catch! Yes, you're right. I'll correct it.